Challo/Challo/Modules/Common/Bookings/BookingRepository.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  BookingRepository.swift
+//  Challo
+//
+//  Created by Tan Le Yang on 30/3/21.
+//
+
+import Foundation
+
+class BookingRepository: Repository<Booking>, BookingRepositoryProtocol {
```

Comment:
I think composition of Repository<Booking> is better. Easier to unit test this class via dependency injection.

Reply:
Thats what I did for my persistence repository too.